+neurostim/stimulus.m

```diff
@@ -388,7 +392,7 @@ function baseBeforeFrame(s)
             %% RSVP mode
             %   Update parameter values if necesssary
             if s.rsvp.active && s.flags.on

```

Comment:
Could/should this `if` block be moved into one directly above to save additional check of `s.flags.on`?

Reply:
True. I moved it directly below.

Reply:
Correction, I moved it up.  updateRSVP can change the flags.on from true to false, so it cannot be moved passed the s.flags.on below.

I moved in the if block where flags.on can be true (it still has to be checked though). Saving some evaluation on s.flags.on == false frames.